src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,446 @@
 ---
-title: Request
+title: Request endpoints
 layout: reference
 ---
 
-# Request
+# Request endpoints
+
+- [Resources](#resource)
+  - [Request](#request-resource)
+  - [Workflow](#workflow-resource)
+  - [Expense](#expense-resource)
+  - [Travel Agency](#travel-agency-resource)
+- [Data Format](#data-format)
+  - [Request](#request)
+  - [ApprovalStatus](#approvalstatus)
+  - [Customfield](#customfield)
+  - [Location](#location)
+  - [Amount](#amount)
+  - [ResourceLink](#resourcelink)
+  - [RequestLink](#requestlink)
+  - [Link](#link)
+  - [Expense](#expense)
+  - [Allocation](#allocation)
+  - [Exchange Rate](#exchangerate)
+  - [Expense Type](#expensetype)
+  - [Travel Allowance](#travelallowance)
+  - [Trip Data](#tripdata)
+  - [Vendor](#vendor)
+  - [Segment Leg](#segmentleg)
+  - [Segment Type](#segmenttype)
+
+
+### <a name="resource"></a>Resources
+Manage documents used for pre-spend authorizations within Concur Request.
+
+#### <a name="request-resource"></a>Request
+- __Create a new request__
+
+POST /v4/requests
+
+*Parameters:*
+| Name | Parameter Type | Data Type | Description |
```

Comment:
The displaying of every table does not show with different lines. is it linked to the display on GitHub or is there another way to create a "clean" table with for this example 4 columns and 2 lines?